src/main/java/com/glencoesoftware/omero/ms/image/region/ImageRegionMicroserviceVerticle.java

```diff
@@ -137,6 +167,29 @@ public void deploy(JsonObject config, Future<Void> future) {
 
         HttpServer server = vertx.createHttpServer();
         Router router = Router.router(vertx);
+      
+        JsonObject prometheusAuth = config.getJsonObject("prometheus_auth");
+        if(prometheusAuth != null) {
+            String prometheus_username = prometheusAuth.getString("username");
```

Comment:
I don't think you need to go to this level with the checks. Use `getString(<key>, <default>)` with an empty string default and go with that if `prometheusAuth` is available.